db/data_migrations/20230601130306_convert_to_active_storage.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require "logger"
+
+class ConvertToActiveStorage < ActiveRecord::DataMigration
```

Comment:
We seems to be making this unnecessarily complicated, we don't need to handle signals manually. Just store the identifier of the last processed item in redis,

Reply:
Added 👍, I have still not allowed interrupt during the profile_picture migration, so it wont stop until it is finished. Since there wont be a lot of images to migrate for avatar and migration is real fast, takes about 10s for 500 images on my system..